example-conf/mesos-dns/mesos-site.xml

```diff
@@ -43,7 +43,7 @@
 
   <property>
     <name>mesos.hdfs.zkfc.ha.zookeeper.quorum</name>
-    <value>master0.mesos:2181,master1.mesos:2181,master2.mesos:2181</value>
```

Comment:
Hey guys,
Sorry for poking you here (couldn't find other channel/contact point). Hope that's okay?
Thanks for working on this. Skimming through it, looks like you guys are working on starting the HDFS processes on mesos. Could you please tell how are you working on to providing resilience in case of slave failure (in case if a mesos slave task dies, etc), given that https://issues.apache.org/jira/browse/MESOS-1554 is still in progress. Or, does this work stand on top of the 1554?
Thanks,
Himanshu